src/backend/Csrs.Interfaces.SharePoint/SharePointFileManager.cs

```diff
@@ -872,10 +928,77 @@ public async Task<string> UploadFile(string fileName, string listTitle, string f
 
                 // make the request.
                 var response = await _Client.SendAsync(endpointRequest);
+                var streamData = response.Content.ReadAsStringAsync().Result;
+
+                var listItemData = JsonConvert.DeserializeObject<AddFileResponse>(streamData);
 
                 if (response.StatusCode == HttpStatusCode.OK)
                 {
                     result = fileName;
+
+                    return await UpdateListItemFields(listItemData, listTitle, contentType, fileName); ;
+                }
+                else
+                {
+                    string _responseContent = null;
+                    var ex = new SharePointRestException(string.Format("Operation returned an invalid status code '{0}'", response.StatusCode));
```

Comment:
Use string interpolation

Reply:
changed all the SharePointRestException to use string interpolation